code401challenges/src/test/java/code401challenges/ArrayShiftTest.java

```diff
@@ -0,0 +1,27 @@
+package code401challenges;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ArrayShiftTest {
+
+    @Test
+    public void insertShiftArray() {
+        ArrayShift test1 = new ArrayShift();
+        ArrayShift test2 = new ArrayShift();
+
+        int[] input1 = new int[]{1, 2, 3, 4, 10, 11};
+        int[] expected1 = new int[]{1, 2, 3, 99, 4, 10, 11};
+
+        int[] input2 = new int[]{1, 2, 3, 4, 10, 11, 12};
+        int[] expected2 = new int[]{1, 2, 3, 4, 77, 10, 11, 12};
+
+        int[] result1 = test1.insertShiftArray(input1, 99);
+        Assert.assertArrayEquals(expected1, result1);
+
+        int[] result2 = test1.insertShiftArray(input2, 77);
+        Assert.assertArrayEquals(expected2, result2);
```

Comment:
I like the idea of doing two tests, however, these should be split into their own methods.

public void insertShiftArrayEven() {
//do stuff
}

public void insertShiftArrayOdd() {
//do stuff
}